### PR TITLE
[logging] adding security logging to state-sync

### PR DESCRIPTION
**Pull request by mimoo**

### CODE REVIEW

File: common/logger/src/security.rs [hunk 1]
```diff
@@ -62,11 +62,11 @@ pub mod security_events {
     /// A block being committed or executed is invalid
     pub const INVALID_BLOCK: &str = "InvalidBlock";
 
-    // Executor
-    // --------
+    // State-Sync
+    // ----------
 
-    /// Executor received an invalid transactions chunk
-    pub const INVALID_CHUNK_EXECUTOR: &str = "InvalidChunkExecutor";
```
Comment (mimoo) [inline common/logger/src/security.rs:69]: I don't think it makes sense to log anything security in executor, as we don't know who sent us the malicious payload there. Better to log things on the client side (state-sync or consensus)

Comment (gregnazario) [reply common/logger/src/security.rs:69]: I'm wondering why is an invalid chunk a security log?

Comment (mimoo) [reply common/logger/src/security.rs:69]: I should probably document this better, security logs are for detection of malicious behaviors by other validators. This pretty much tells us that a validator full node has sent us a chunk of transaction that is incorrect (this should not happen in practice unless someone is trying to mess with us).

**Pull request closed without merging**